pkg/cmd/dotgit/find.go

```diff
@@ -0,0 +1,99 @@
+package dotgit
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/MaineK00n/vuls-data-update/pkg/dotgit/find"
+)
+
+func newCmdFind() *cobra.Command {
+	options := &struct {
+		treeish string
+
+		format formatType
+	}{
+		treeish: "HEAD",
```

Comment:
main is better?

Reply:
changed: https://github.com/MaineK00n/vuls-data-update/compare/e207dd2cc1ef8bd1cc85409c6c4b090ad1a97730..030a8396e80a472366fc0bec089b9c1cbbf9b413